app/src/androidTest/java/com/karhoo/karhootraveller/splash/SplashTests.kt

```diff
@@ -306,15 +305,15 @@ class SplashTests : Launch {
             reverseGeocodeResponse(HTTP_OK, REVERSE_GEO_SUCCESS)
         }
         splash(this) {
-            sleep(MEDIUM)
+            shortSleep()
```

Comment:
should we use `mediumSleep()`

Reply:
I've tested it and a short sleep is enough